app/controllers/schools_controller.rb

```diff
@@ -34,6 +34,9 @@ def school_administrator_dashboard
       individual_student_dashboard_data(student)
     end
 
+    dashboard_students = students_for_dashboard(@school).includes([homeroom: :educator], :dashboard_absences, :event_notes, :dashboard_tardies, :discipline_incidents)
+                                                        .map { |student| individual_student_dashboard_data(student) }
```

Comment:
Let's talk about this; it impacts the existing pages load more data that users can't yet see, and those pages are already slow enough we'll need to optimize this.  I can look at this more next week with you and figure out what design makes sense, and then I can take care of the server-side optimizations since it's it's hard to optimize for production data without being able to measure :)

Reply:
@kevinrobinson I'd love to go over this. I've tried to tighten up the queries, but probably the way to go is to separate these based on the dashboard the user is looking for.

Reply:
Sure, I'll look at this this morning and send you the other notes that came up Friday.